pygmt/figure.py

```diff
@@ -489,26 +480,26 @@ def show(
                     "'notebook', 'none' or None."
                 )
 
-    def _preview(self, fmt, dpi, as_bytes=False, **kwargs):
+    def _preview(self, fmt: str, dpi: int, as_bytes: bool = False, **kwargs):
         """
         Grab a preview of the figure.
 
         Parameters
         ----------
-        fmt : str
-            The image format. Can be any extension that
-            :meth:`pygmt.Figure.savefig` recognizes.
-        dpi : int
+        fmt
+            The image format. Can be any extension that :meth:`pygmt.Figure.savefig`
+            recognizes.
+        dpi
             The image resolution (dots per inch).
-        as_bytes : bool
-            If ``True``, will load the image as a bytes string and return that
-            instead of the file name.
+        as_bytes
+            If ``True``, will load the binary contents of the image as a bytes object,
+            and return that instead of the file name.
 
         Returns
         -------
-        preview : str or bytes
-            If ``as_bytes=False``, this is the file name of the preview image
-            file. Else, it is the file content loaded as a bytes string.
+        preview
+            If ``as_bytes=False``, this is the file name of the preview image file.
+            Otherwise, it is the file content loaded as a bytes object.
```

Comment:
```suggestion
        preview
            If ``as_bytes = False``, this is the file name of the preview image file.
            Otherwise, it is the file content loaded as a bytes object.
```